### ISSUE TITLE
[YSQL] Separate out metrics for BEGIN, COMMIT, ROLLBACK statements from the OTHER category

### ISSUE THREAD

**Issue by kmuthukk** [yugabyte/yugabyte-db#6486]
Today the top-level YSQL statement metrics at the TServer level are categorized as:

For example:
http://yb-tserver:13000/metrics

See output below. It'll be good to break out from the OtherStmts category, separate metrics for Begin/Commit/Rollback transaction statements.  [For PREPARE statements, the handling is a bit different. So will fork that into its own issue.]

```
  {
        "type": "server",
        "id": "yb.ysqlserver",
        "metrics": [
            {
                "name": "handler_latency_yb_ysqlserver_SQLProcessor_SelectStmt",
                "count": 205539578,
                "sum": 1239986847119
            },
            {
                "name": "handler_latency_yb_ysqlserver_SQLProcessor_InsertStmt",
                "count": 116057923,
                "sum": 827644852030
            },
            {
                "name": "handler_latency_yb_ysqlserver_SQLProcessor_DeleteStmt",
                "count": 0,
                "sum": 0
            },
            {
                "name": "handler_latency_yb_ysqlserver_SQLProcessor_UpdateStmt",
                "count": 0,
                "sum": 0
            },
            {
                "name": "handler_latency_yb_ysqlserver_SQLProcessor_OtherStmts",
                "count": 3450,
                "sum": 27716148
            },
            {
                "name": "handler_latency_yb_ysqlserver_SQLProcessor_Transactions",
                "count": 321593501,
                "sum": 2067614691134
            },
            {
                "name": "handler_latency_yb_ysqlserver_SQLProcessor_AggregatePushdowns",
                "count": 0,
                "sum": 0
            }
        ]
    }
```

**Comment by VishnuK007** [comment]
I have a patch that I would like to submit for this issue. Can you please assign it to me if no one has started working on it yet?

**Comment by rkarthik007** [comment]
Thanks for your interest @VishnuK007 🔥 !

**Comment by m-iancu** [comment]
Fixed by e297413.
